doc/content/the-things-stack/migrating/migrating-between-tts-distributions/establish-new-session/_index.md

```diff
@@ -50,7 +53,7 @@ To export device using the [migration tool]({{< ref "/the-things-stack/migrating
 ttn-lw-migrate device --source tts 'my-device' --tts.no-session > devices.json
 ```
 
-Next, you need to import the `devices.json` file in your {{% tts %}} Cloud application. See instructions on how to [Import End Devices in {{% tts %}}]({{< ref "/the-things-stack/migrating/import-devices" >}}). Keep in mind that if you are using the CLI to import devices, you have to configure it to connect to {{% tts %}} Cloud. See [Configuring the CLI]({{< ref "/the-things-stack/interact/cli/configuring-cli" >}}) guide for instructions.
+Next, you need to import the `devices.json` file in your {{% tts %}} Cloud application. See instructions on how to [Import End Devices in {{% tts %}}]({{< ref "/devices/adding-devices/adding-devices-in-bulk" >}}). Keep in mind that if you are using the CLI to import devices, you have to configure it to connect to {{% tts %}} Cloud. See [Configuring the CLI]({{< ref "/the-things-stack/interact/cli/configuring-cli" >}}) guide for instructions.
```

Comment:
```suggestion
Next, you need to import the `devices.json` file in your {{% tts %}} Cloud application. See instructions on how to [import end devices in {{% tts %}}]({{< ref "/devices/adding-devices/adding-devices-in-bulk" >}}). Keep in mind that if you are using the CLI to import devices, you have to configure it to connect to {{% tts %}} Cloud. See [Configuring the CLI]({{< ref "/the-things-stack/interact/cli/configuring-cli" >}}) guide for instructions.
```

Reply:
Just because the section isn't called "Import End Devices" anymore; if you change it, pls look for other occurrences